libs/wallet/src/use-eager-connect.ts

```diff
@@ -11,44 +11,39 @@ export function useEagerConnect(Connectors: {
 
   useEffect(() => {
     const attemptConnect = async () => {
-      const cfg = LocalStorage.getItem(WALLET_CONFIG);
-      let cfgObj: { connector: 'rest'; token: string } | null;
-
-      try {
-        cfgObj = cfg ? JSON.parse(cfg) : null;
-      } catch {
-        cfgObj = null;
-      }
+      const cfg = getConfig();
 
       // No stored config, or config was malformed
-      if (!cfgObj || !cfgObj.connector) {
+      if (!cfg || !cfg.connector) {
         setConnecting(false);
         return;
       }
 
       // Use the connector string in local storage to find the right connector to auto
       // connect to
-      const connector = Connectors[cfgObj.connector];
+      const connector = Connectors[cfg.connector];
 
       // Developer hasn't provided this connector
       if (!connector) {
         setConnecting(false);
         console.error(
-          `Can't eager connect, connector: ${cfgObj.connector} not found`
+          `Can't eager connect, connector: ${cfg.connector} not found`
         );
         return;
       }
 
       try {
-        await connect(Connectors[cfgObj.connector]);
+        await connect(Connectors[cfg.connector]);
       } catch {
-        console.warn(`Failed to connect with connector: ${cfgObj.connector}`);
+        console.warn(`Failed to connect with connector: ${cfg.connector}`);
```

Comment:
Chore (blocking): translation needed

Reply:
Hm not sure if we need translations for console warnings. For me only things that are shown in the UI should need them.